frontend/src/app/ui/material-theme/gendalf-theme.scss

```diff
@@ -0,0 +1,26 @@
+@import '~@angular/material/theming';
```

Comment:
please merge latest master inside your branch and check with @vladimir-sviridenko that this update does not affect his changes in `button` component

Reply:
also please move this to app root folder, near `styles.scss`